src/my_project/tagging/tagging_controller.py

```
@@ -78,188 +80,124 @@ def cover_getter(self):
-        self.search_results = IdSet[ComicVineSearchStruct]()
-        for q in queries:
-            if q == "":
-                continue
-            logger.info(f"Query: {q}")
-            self.http.build_url_search(q)
-            results = self.http.search_get_request()
-            self.search_validator = SearchResponseValidator(results.results, self.data)
+        logger.info(f"There are {len(self.search_results)} results returned.")
+        filtered_results = self.search_validator.filter_search_results()
+        if len(filtered_results) == 0:
+            raise RuntimeError("NO RESULTS")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)